pkg/stanza/flush/flush.go

```diff
@@ -6,8 +6,12 @@ package flush // import "github.com/open-telemetry/opentelemetry-collector-contr
 import (
 	"bufio"
 	"time"
+
+	"github.com/tilinna/clock"
```

Comment:
I haven't looked into the details much yet but this library only has a dozen stars and hasn't been updated in years. I did a quick search and found https://github.com/jonboulle/clockwork which may be a bit safer to standardize on, if it can be used for the same.

Reply:
Good point! The `github.com/tilinna/clock` library is already used at the [`spanmetricsconnector`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d6be16b6bc3b3c81f67821974c1d84eb73902e3f/connector/spanmetricsconnector/factory.go#L40) though, but maybe it can also be replaced there (https://github.com/jonboulle/clockwork/blob/master/context.go#L25).

Reply:
Either way then, but I see you already changed it. 👍